platformio.ini

```diff
@@ -286,15 +262,15 @@ lib_deps =
 
 [esp32c3]
 ;; generic definitions for all ESP32-C3 boards
-platform = espressif32@5.3.0
-platform_packages =
+platform = ${esp32.platform}
+platform_packages = ${esp32.platform_packages}
 build_flags = -g
-  -DARDUINO_ARCH_ESP32
-  -DARDUINO_ARCH_ESP32C3
-  -DCONFIG_IDF_TARGET_ESP32C3=1
+  -D ARDUINO_ARCH_ESP32
+  -D ARDUINO_ARCH_ESP32C3
+  -D FASTLED_ALL_PINS_HARDWARE_SPI
```

Comment:
and `-D FASTLED_ESP32_SPI_BUS=HSPI` in this case

Reply:
this plus `-D FASTLED_ESP32_SPI_BUS=FSPI` for the `lolin_s2_mini` target gets past the build errors.